main.go

```diff
@@ -116,3 +129,50 @@ Arguments:
 		helpTab,
 	)
 }
+
+// This section translates a windows path such as "C:\Users\user" to "/Users/user"
+// and removes the default paths for mingw and git4windows to issues when you
+// specify a file path to start code-server in.
+func gitbashWindowsDir(dir string) string {
+	if dir == "~" { //Special case
```

Comment:
Comments should be above, not beside their respective if. In this case the comment isn't really necessary since it's clear what it does.

Reply:
I have sent a email to eargollo, the original maker of pretty much everything in this latest push inside the "manually merged" commit, he has been silent for a while.

Reply:
From what i can interpet, 
this checks if `dir = "~"`,
if so, return `dir ="~/"` and ingore the rest of the function, skipping to the end as it is unnecessary at that moment in time